op-node/rollup/types.go

```diff
@@ -686,6 +686,7 @@ func (c *Config) LogDescription(log log.Logger, l2Chains map[string]string) {
 		"interop_time", fmtForkTimeOrUnset(c.InteropTime),
 		"alt_da", c.AltDAConfig != nil,
 		"l2_blob_config", c.L2BlobConfig != nil,
+		"use_inbox_contract", c.UseInboxContract(),
```

Comment:
If not necessary, I would suggest avoiding modifications to the upstream code.

Reply:
I think showing this log would be better.